### PR TITLE
[WIP] Studio resizable UI

### PR DESCRIPTION
**Pull request by chrisplatorres**
This PR add resizing functionality to the Studio UI.

draggy

### CODE REVIEW

**Comment by chrisplatorres** [comment]
There are a few things that need to be done here still:
- resizing one menu also updates other UI sizes
- should more of the UI be resizable? It seems only the right sidebar needs to be resized at the moment

**Comment by modulesio** [comment]
> should more of the UI be resizable? It seems only the right sidebar needs to be resized at the moment

Probably the bottom console bar too, though it should also have a hide ability.

**Comment by chrisplatorres** [comment]
> Probably the bottom console bar too, though it should also have a hide ability.

I'll add that into this PR, the console will be hidden by default, but when it is visible it is resizable.

**Comment by chrisplatorres** [comment]
Cloning and deleting tabs, as well as resizing works as expected. Going to fix up the UI so that..

click `...`
then there is a  menu similar to above attr one, but is smaller, directly under the site, and has options for:
- clone 
- delete
- edit (auto-detect site and open in an three/aframe/babylon editor or if not able to be detected, open in a text editor)

**Comment by modulesio** [comment]
Awesome!

The editor break-out/auto-detect sounds like it's much bigger than the rest. Did you want to merge this one first and then continue with a new PR?

**Comment by chrisplatorres** [comment]
Yes, and I agree.. the "editing a site" part of Studio should be in a separate PR.

**Comment by chrisplatorres** [comment]
This can now be merged and will continue work through other branches.

st06Capture

File: examples/launcher/src/components/Dom.jsx [hunk 1]
```diff
@@ -229,14 +269,51 @@ class DomDetail extends React.Component {
   /* constructor(props) {
     super(props);
   } */
+  toggleOpen() {
```
Comment (modulesio) [inline examples/launcher/src/components/Dom.jsx]: If [`DomDetail` is removed](https://github.com/exokitxr/exokit/pull/1076/files#diff-43a85fe06ede43ea8d3091dd359ef2b7L102), why is this being edited as well?

File: examples/launcher/src/components/Dom.jsx [hunk 2]
```diff
@@ -99,7 +97,6 @@ class Dom extends React.Component {
     return (
       <div className="Dom">
         <DomList root={this.state.root} selectEl={this.state.selectEl} onClick={el => this.onClick(el)} />
```
Comment (modulesio) [inline examples/launcher/src/components/Dom.jsx]: If the intent is to re-add `DomDetail` later, perhaps just comment it?

File: examples/launcher/src/components/Dom.jsx [hunk 3]
```diff
@@ -189,17 +187,59 @@ class DomItem extends React.Component {
...
+      dropdownOpen: !this.state.dropdownOpen,
+    });
+
+  }
+
+  cloneTab() {
+    const {el} = this.props;
+    let url;
+
+    el.attrs.map(attr => (attr.name === "src") ? url = attr.value : null);
```
[the remark this answers is not in the document]
Comment (chrisplatorres) [reply examples/launcher/src/components/Dom.jsx]: b5d4ae4

File: examples/launcher/src/components/Dom.jsx (hunk 2 above)
Comment (chrisplatorres) [reply examples/launcher/src/components/Dom.jsx]: 29588b7

File: examples/launcher/src/components/Dom.jsx (hunk 1 above)
Comment (chrisplatorres) [reply examples/launcher/src/components/Dom.jsx]: reverted in 
ad9ecad

**Comment by chrisplatorres** [comment]
Screenshot for style referencing:
stu1Capture

**Pull request merged**